recipes-containers/podman/podman_%.bbappend

```diff
@@ -21,4 +21,3 @@ RDEPENDS:${PN} += "pasta shadow"
 
 FILES:${PN} += "${sysconfdir}/init.d/podman-init \
                 ${sysconfdir}/containers/containers.conf"
-                
```

Comment:
👀

Reply:
But you are removing the newline at the end of the file here. That is why I was confused 🤔 
My IDE also removes the newline at the end of each file but I add them back.